File: Modul 7/Soal 3/PRAK603-2210817310006-Ahmad Muhsin Zein.c
#include <stdio.h>

int main()
{
    int a1,a2,i,j,x,hasil;
    scanf("%d %d", &a1, &a2);
    int y1[a1], y2[a2];
    if(a1==a2){
        for(int i = 0; i < a1; i ++){
        scanf("%d",&x);
        y1[i]=x;
        }
        for(int i = 0; i < a2; i ++){
        scanf("%d",&x);
        y2[i]=x;
        }
        for(int i = 0; i < a1; i ++){
        hasil = y1[i]*y2[i];
        printf(" %d ",hasil);
        }
    }
    else{
        printf("Jumlah Tidak Sama");
    }
}

File: Modul 7/Soal 3/PRAK603-2210817310006-Ahmad Muhsin Zein.py
a,b = map(int, input().split())
matrix = [];matrix2 = []
if a == b:
    x = list(map(int, input().split()))
    x1 = list(map(int, input().split()))
    for i in range(a):
        matrix.append(x[i])
    for i in range(b):
        matrix2.append(x1[i])
    for i in range(a) :
        hasil = matrix[i]*matrix2[i]
        print(hasil, end = " ")
    print()
else :
    print('Jumlah Tidak Sama')